hash/bucketer_test.go

```diff
@@ -101,34 +101,47 @@ func TestBucketSetUpdate(t *testing.T) {
 	}
 }
 
-func TestBucketSetNewBucket(t *testing.T) {
+func TestBucketSetBuckets(t *testing.T) {
 	bs := NewBucketSet(buckets)
-	_, err := bs.NewBucket(thisBucket)
-	if err != nil {
-		t.Error("NewBucket = ", err)
+	bkts := bs.Buckets()
+
+	// Sorted
+	want := []string{"aguacero", "chaparrón", "chubasco", "monsoon"}
+	got := make([]string, 4)
+	for i, b := range bkts {
+		got[i] = b.Name()
 	}
-	_, err = bs.NewBucket("giboulée")
-	if err == nil {
-		t.Error("NewBucket unexpectedly succeeded")
+
+	if !reflect.DeepEqual(got, want) {
+		t.Errorf("got %v, want %v", got, want)
```

Comment:
```suggestion
		t.Errorf("Got %v, want %v", got, want)
```
also print diff?

Reply:
I think it's easy to get the diff from the output as the array isn't too big.

Reply:
🤷 why make me work, when we can make the prow do the work :)

Reply:
fine. done